app/src/main/res/xml/todo_widget_info.xml

```diff
@@ -15,5 +15,4 @@
     android:description="@string/todo_widget_desc"
     android:widgetCategory="home_screen"
     android:previewImage="@drawable/widget_example_todo"
-    android:previewLayout="@layout/widget_todo_wrapper_dynamiccolor"
```

Comment:
Does Glance not support a preview layout? That would be a shame, it's nice to have the widget previews translated and adopt your theme's color.

Reply:
If I'm not wrong the only way to do it is to have a XML version and TBH I didn't want to do that.

Reply:
Actually I found this https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:glance/glance-appwidget-preview/src/main/java/androidx/glance/appwidget/preview/GlanceAppWidgetViewAdapter.kt while making the `bundle` I was wondering why there was 2 preview libraries.

I'm gonna try it out.